utils/warnings.ml

```diff
@@ -775,6 +779,8 @@ let descriptions =
    64, "-unsafe used with a preprocessor returning a syntax tree";
    65, "Type declaration defining a new '()' constructor";
    66, "Unused open! statement";
+   (* 67, "ask octachron" *)
```

Comment:
Indeed, the short description for the `unused functor argument` warning is missing. If you want to add the missing text, it would make for a fine separate PR. Otherwise, myself (or @trefis) will fix the issue.

Reply:
Okay I ll add it.